src/Sulu/Bundle/TestBundle/Testing/CreateClientTrait.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\TestBundle\Testing;
+
+use Sulu\Component\HttpKernel\SuluKernel;
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
+use Symfony\Component\BrowserKit\Client;
+
+trait CreateClientTrait
```

Comment:
Why don't we need all the environment information anymore in these functions?

Reply:
they matches the default environment variables so not needed to have them also here